api/mysagw/identity/management/commands/upload_labels_template.py

```diff
@@ -0,0 +1,54 @@
+from argparse import ArgumentTypeError
+from pathlib import Path
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from requests import HTTPError
+
+from mysagw.identity.dms_client import DMSClient
+
+TEMPLATE_PATH = Path(__file__).parent.parent.parent.absolute() / "templates"
+
+
+class Command(BaseCommand):
+    help = "Upload template for printing address labels"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.client = DMSClient()
+
+    def template_file(self, value):
+        template = TEMPLATE_PATH / f"{value}.docx"
```

Comment:
I'd suggest accepting a complete path instead of the name-without-extension only. This would allow tab completion and would also show the reader of the code what it's doing, without having to look at the code to see where the template is loaded from

Reply:
Done